lib/rubocop/cop/rspec/leaky_constant_declaration.rb

```diff
@@ -17,6 +17,8 @@ module RSpec
       # Anonymous classes are fine, since they don't result in global
       # namespace name clashes.
       #
+      # @see https://relishapp.com/rspec/rspec-mocks/docs/mutating-constants
```

Comment:
While I like this change (extract duplication), it seems the link doesn’t make it to the manual now.

Reply:
It seems to be quite common:
```
lib/rubocop/cop/rspec/subject_stub.rb|8| 10:      # @see https://robots.thoughtbot.com/don-t-stub-the-system-under-test
lib/rubocop/cop/rspec/leaky_constant_declaration.rb|20| 10:      # @see https://relishapp.com/rspec/rspec-mocks/docs/mutating-constants
lib/rubocop/cop/rspec/context_wording.rb|8| 10:      # @see https://github.com/reachlocal/rspec-style-guide#context-descriptions
lib/rubocop/cop/rspec/context_wording.rb|9| 10:      # @see http://www.betterspecs.org/#contexts
lib/rubocop/cop/rspec/example_wording.rb|10| 10:      # @see http://betterspecs.org/#should
lib/rubocop/cop/rspec/verified_doubles.rb|8| 10:      # @see https://relishapp.com/rspec/rspec-mocks/docs/verifying-doubles
lib/rubocop/cop/rspec/describe_symbol.rb|19| 10:      # @see https://github.com/rspec/rspec-core/issues/1610
lib/rubocop/cop/rspec/multiple_expectations.rb|8| 10:      # @see http://betterspecs.org/#single Single expectation test
```

Do you mind if I file a ticket and we could either fix code documentation to use plain "See: ", or fix documentation generator to include `see` to the manual?

Reply:
Good idea.

Reply:
Filed #772.